examples/pom.xml

```diff
@@ -20,14 +20,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.apache.james</groupId>
-        <artifactId>james-project</artifactId>
-        <version>3.6.0-SNAPSHOT</version>
-    </parent>
```

Comment:
We at least need to depend on the root Apache POM as a parent .

See https://infra.apache.org/publishing-maven-artifacts.html

Reply:
Good point. Made root apache pom as parent.